src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,27 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+
+interface SingleMovie {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+}
+
+type Mov = {
+  list: SingleMovie[];
+};
+
+export const MoviesList: React.FC<Mov> = ({ list }) => (
+
+  <div className="movies">
+    {list.map((el : SingleMovie) => (
+      <div key={el.imdbId} data-cy="card" className="card">
+        <MovieCard {...el} />
```

Comment:
We don't  allowed to use `...` in passing arguments.

Reply:
Agree, fix it
You should pass correct values to specified properties to avoid errors